lib/bolt/config/options.rb

```diff
@@ -58,11 +58,33 @@ module Options
                        "plan function or the `bolt apply` command.",
           type: Hash,
           properties: {
+            "evaltrace" => {
+              description: "Whether each resource should log when it is being evaluated. This allows "\
+                           "you to interactively see exactly what is being done.",
```

Comment:
```suggestion
```
This doesn't stream output, so I'm not sure including "interactive" is really useful. I think just the first sentence is good.